tests/components/freebox/test_alarm_control_panel.py

```diff
@@ -0,0 +1,128 @@
+"""Tests for the Freebox sensors."""
+from copy import deepcopy
+from unittest.mock import Mock
+
+from freezegun.api import FrozenDateTimeFactory
+import pytest
+
+from homeassistant.components.alarm_control_panel import (
+    DOMAIN as ALARM_CONTROL_PANEL,
+    AlarmControlPanelEntityFeature,
+)
+from homeassistant.components.freebox import SCAN_INTERVAL
+from homeassistant.const import (
+    SERVICE_ALARM_ARM_AWAY,
+    SERVICE_ALARM_ARM_CUSTOM_BYPASS,
+    SERVICE_ALARM_ARM_HOME,
+    SERVICE_ALARM_ARM_NIGHT,
+    SERVICE_ALARM_ARM_VACATION,
+    SERVICE_ALARM_DISARM,
+    SERVICE_ALARM_TRIGGER,
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_CUSTOM_BYPASS,
+    STATE_ALARM_ARMED_HOME,
+    STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_ARMED_VACATION,
+    STATE_ALARM_DISARMED,
+    STATE_ALARM_TRIGGERED,
+)
+from homeassistant.core import HomeAssistant, State
+from homeassistant.helpers.state import async_reproduce_state
+
+from .common import setup_platform
+from .const import DATA_HOME_ALARM_GET_VALUES
+
+from tests.common import async_fire_time_changed, async_mock_service
+
+
+async def test_panel(
+    hass: HomeAssistant, freezer: FrozenDateTimeFactory, router: Mock
+) -> None:
+    """Test home binary sensors."""
+    await setup_platform(hass, ALARM_CONTROL_PANEL)
+
+    # Initial state
+    assert hass.states.get("alarm_control_panel.systeme_d_alarme").state == "disarmed"
+    assert (
+        hass.states.get("alarm_control_panel.systeme_d_alarme").attributes[
+            "supported_features"
+        ]
+        == AlarmControlPanelEntityFeature.ARM_AWAY
+    )
+    assert (
+        hass.states.get("alarm_control_panel.systeme_d_alarme").attributes["pin"]
+        == "0000"
+    )
```

Comment:
This is the alarm code that shouldn't be here: https://github.com/home-assistant/core/pull/102607#discussion_r1373034946